### PR TITLE
Update Migration doc

### PR DESCRIPTION
**Pull request by cacheung**
## Screenshots (if appropriate):
Update Migration doc.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by yangyansong-adbe — APPROVED** [review]

File: docs/migration.md [hunk 1]
```diff
@@ -126,10 +155,14 @@ public class MainApplication extends Application implements ReactApplication {
...
         AEPMobileLifecycle.class,
         AEPMobileSignal.class,
-        AEPMobileIdentity.class,
-        AEPMobileUserProfile.class,
+        AEPMobileTarget.class,
+        AEPMobileCampaignClassic.class,
+        AEPMobilePlaces.class,
+        AEPMobileAssurance.class,
     ] completion:^{
     [AEPMobileCore lifecycleStart:@{@"contextDataKey": @"contextDataVal"}];
```
Comment (praveek) [inline docs/migration.md:167]: We need to check if the app is not in background before calling lifecycleStart.
Can you also update the other places with similar logic?

**Review by praveek — CHANGES_REQUESTED** [review]

File: docs/migration.md [hunk 2]
```diff
@@ -11,30 +11,31 @@ Open your app's package.json file and replace the ACP-prefixed packages with the
...
+| @adobe/react-native-acpplaces | @adobe/react-native-aepplaces |
 | @adobe/react-native-acpplaces-monitor | NA |
 | @adobe/react-native-acpanalytics | NA |
 | @adobe/react-native-acpmedia | NA |
 | @adobe/react-native-acpaudience | NA |
-| @adobe/react-native-acptarget | NA |
-| @adobe/react-native-acpcampaign | NA |
-| @adobe/react-native-aepassurance:2.x (compatible with ACP libraries) | @adobe/react-native-aepassurance:3.x (compatible with AEP libraries)|
+| @adobe/react-native-acptarget | @adobe/react-native-aeptarget |
+| @adobe/react-native-acpcampaign | @adobe/react-native-aepcampaignclassic |
```
Comment (rymorale) [inline docs/migration.md:31]: the react-native-acpcampaign wrapper is for campaign standard. the aepcampaignclassic is brand new and does not have an existing acp version. the aepcampaignstandard wrapper is to be done.

File: packages/assurance/README.md [hunk 3]
```diff
@@ -40,15 +40,19 @@ iOS
...
+        [AEPMobileCore lifecycleStart:nil];
+      }
   }];
   return YES;
 }
-
 ```
 
-To connect to a griffon session by scanning the QR code. Follow [Apple developer](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) documentation to set custom URL scheme for your application. Finally, implement the `openURL` delegate method and pass the deeplink URL to startSessionWithURL API.
+To connect to an Assurance session by scanning the QR code. Follow [Apple developer](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) documentation to set custom URL scheme for your application. Finally, implement the `openURL` delegate method and pass the deeplink URL to startSessionWithURL API.
```
Comment (rymorale) [inline packages/assurance/README.md:55]: "To connect to an Assurance session by scanning the QR code, follow ..."

File: packages/assurance/README.md [hunk 4]
```diff
@@ -82,7 +86,7 @@ public class MainApplication extends Application implements ReactApplication {
 }
 ```
 
-To connect to a griffon session by scanning the QR code. Follow the [Android documentation](https://developer.android.com/training/app-links/deep-linking) on information about how to setup a deeplink.
+To connect to an Assurance session by scanning the QR code. Follow the [Android documentation](https://developer.android.com/training/app-links/deep-linking) on information about how to setup a deeplink.
```
Comment (rymorale) [inline packages/assurance/README.md:89]: "To connect to an Assurance session by scanning the QR code, follow the [Android documentation](https://developer.android.com/training/app-links/deep-linking) for more information about how to setup a deeplink."

File: docs/migration.md [hunk 5]
```diff
@@ -28,11 +28,9 @@ At this time, the following ACP-prefix libraries can be switched out with their
 | @adobe/react-native-acpmedia | NA |
 | @adobe/react-native-acpaudience | NA |
 | @adobe/react-native-acptarget | @adobe/react-native-aeptarget |
-| @adobe/react-native-acpcampaign | @adobe/react-native-aepcampaignclassic |
```
Comment (praveek) [inline docs/migration.md]: Can you add a line for campaignclassic?

**Review by praveek — APPROVED** [review]

**Pull request merged**